### ISSUE TITLE
Special character and diacritic issues on local build

### ISSUE THREAD

**Issue by gleeming** [bcgov/ols-geocoder#131]
Prod CPF and autocomplete seems to handle special characters as intended but the same August data loaded on gazelle is having issues.  Could be a UTF-8/windows oddity going on.  The GeoJSON files look like they have the correct character set.

Examples:
Tête Jaune Cache, BC
Deeder Rd, François Lake, BC

**Comment by cmhodgson** [comment]
For some reason, at least the locality file (I haven't checked others) was being read using the windows default "windows-1212" character set. I'm guessing that the BOM (Byte Order Marker) got removed from the data file(s) at some point - the BOM is only technically necessary for UTF-16 to denote little-/big-endian interpretation of the byte order, but if it is present it can also allow some software to automatically detect that the file is in UTF-8, not just ASCII or an unknown/default charset. I've now modified my readers to force the interpretation of all text files (JSON, TSV, CSV) as UTF-8.

**Comment by cmhodgson** [comment]
I don't see any BOMs on any files old or new so that's not it. I'm guessing this might have changed when we moved to java 11 although I have nothing to back that up. To prevent having to build another version of the code with only this "force UTF-8" change, I have added "-Dfile.encoding=UTF-8" to the java startup options on our tomcat9 service. This causes the default charset to be UTF-8 instead of windows-1212 which then causes the files to be read correctly. The code changes to force reading the input files as UTF-8 will be part of the next release (4.1.0).
